### DISCUSSION TITLE
Discord Server

### DISCUSSION THREAD

**Discussion by guilhermepozo** [dracula/dracula-ui#121] [Ideas] [answered]
Hey guys, im a new Dracula lover and newbie frontend student.

So, im trying to "import" that theme into one Hugo theme and im facing some difficulties, so, I thought, maybe existi a discord server to awser some questions about theme and frontend subjects, existis one? If not, what about create new one?

**Comment by sergiuprt** [comment, marked as answer]
here https://discord.gg/vyA9AdqdbK

**Reply by guilhermepozo** [reply]
ty!
